### ISSUE TITLE
Is the code in 2_logistic_regression.py for softmax regression or logistic regression?

### ISSUE THREAD

**Issue by sunshineatnoon** [Newmu/Theano-Tutorials#9]
I don't see any logistic function code in 2_logistic_regression.py, is this really code for logistic regression?

**Comment by HY-Scatterer** [comment]
the logistic regression is implemented by minimizing cross entropy and using the softmax function as the link function
